openapi/index.yaml

```diff
@@ -338,25 +338,181 @@ paths:
+      summary: Create Service
+      description: |
+        Create a new Service with the attributes provided in the request payload.
+      parameters:
+        - in: body
+          name: body
+          description: The service payload.
+          schema:
+            "$ref": "#/definitions/ServicePayload"
+      responses:
+        "200":
+          description: Service created.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "400":
+          description: Invalid payload.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Unauthorized.
+        "429":
+          description: Too many requests.
+        "500":
+          description: Internal Server Error.
+    get:
+      operationId: getServices
+      summary: Get Services
+      description: |
+        Retrieve a list of services.
+      responses:
+        "200":
+          description: The list of service ids.
+          schema:
+            "$ref": "#/definitions/ServiceIdCollection"
+        "401":
+          description: Unauthorized.
+        "429":
+          description: Too many requests.
+  "/services/{service_id}":
+    parameters:
+      - name: service_id
+        in: path
+        type: string
+        required: true
+        description: The ID of an existing Service.
+    get:
+      operationId: getService
+      summary: Get Service
+      description: |
+        Retrieve a previously created service with the provided service ID.
+      responses:
+        "200":
+          description: Service found.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "401":
+          description: Unauthorized.
+        "404":
+          description: No service found for the provided ID.
+        "429":
+          description: Too many requests.
+    put:
+      operationId: updateService
+      summary: Update Service
+      description: |
+        Update a previously created service with the provided service ID
+      parameters:
+        - in: body
+          name: body
+          description: The service payload.
+          schema:
+            "$ref": "#/definitions/ServicePayload"
+      responses:
+        "200":
+          description: Service updated.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "401":
+          description: Unauthorized.
+        "404":
+          description: No service found for the provided ID.
+        "429":
+          description: Too many requests.
+        "500":
+          description: Internal Server Error.
+  "/services/{service_id}/logo":
+    parameters:
+      - name: service_id
+        in: path
+        type: string
+        required: true
+        description: The ID of a existing Service.
+    put:
+      summary: Upload service logo.
+      description: |
+        Upload a logo for a previously created service with the provided service ID
```

Comment:
these are public APIs, we should be more informative (ie. which kind of data must we send here ? base64 encoded png ?)

Reply:
Fixed in [d1fea39](https://github.com/pagopa/io-functions-services/pull/67/commits/d1fea39496f18158848e087c920b5e5e6c3c0073)